apiary.apib

```diff
@@ -19,6 +19,10 @@ Our API consists of HTTP RPC-style methods, in the form of `https://api.focus.te
 
 We chose this action based approach over the more popular REST, because it enables us to have domain related actions on resources such as `invoices.book`, `timetracking.start` and `timetracking.stop`.
 
+### Transport Security
+
+We expect all network clients to make use of TLS 1.2 when connecting to our infrastructure.
```

Comment:
@stevenn or @YorickH should we mention "at least" here? I think we also support TLS 1.3 already?